minefield_game/board.cpp

```diff
@@ -100,7 +23,7 @@ namespace BoardUtils
 
             for (size_t x = 0; x < board.axisX.size(); ++x)
             {
-                // Chequeo si esta celda es mina del jugador
+                // Check if this cell is/has a player's mine
                 bool isMineForPlayer = false;
                 for (const auto& mine : player.mines)

```

Comment:
#could just write `const Mine& mine`; `auto` and `Mine` have the same amount of characters 😅